index.bs

```diff
@@ -1526,39 +1525,40 @@ In order to report a {{Gamepad/mapping}} of {{GamepadMappingType/xr-standard}} t
   <thead>
     <tr>
       <th>Axes</th>
-      <th><code>xr-standard</code> Location</th>
+      <th><code>xr-standard</code> Binding</th>
       <th>Required</th>
     </tr>
   </thead>
   <tbody>
     <tr>
       <td>axes[0]</td>
-      <td>Primary Touchpad/Joystick X</td>
-      <td>Yes</td>
+      <td>Touchpad X</td>
+      <td>If thumbstick is not present</td>
     </tr>
     <tr>
       <td>axes[1]</td>
-      <td>Primary Touchpad/Joystick Y</td>
-      <td>Yes</td>
+      <td>Touchpad Y</td>
+      <td>If thumbstick is not present</td>
     </tr>
     <tr>
       <td>axes[2]</td>
-      <td>Secondary Touchpad/Joystick X</td>
-      <td>No</td>
+      <td>Thumbstick X</td>
+      <td>If touchpad is not present</td>
     </tr>
     <tr>
       <td>axes[3]</td>
-      <td>Secondary Touchpad/Joystick Y</td>
-      <td>No</td>
+      <td>Thumbstick Y</td>
+      <td>If touchpad is not present</td>
     </tr>
   </tbody>
 </table>
 
 Devices that lack one of the optional inputs listed in the tables above MUST preserve their place in the {{Gamepad/buttons}} or {{Gamepad/axes}} array, reporting a <dfn>placeholder button</dfn> or <dfn>placeholder axis</dfn>, respectively. A [=placeholder button=] MUST report <code>0</code> for {{GamepadButton/value}}, <code>false</code> for {{GamepadButton/pressed}}, and <code>false</code> for {{GamepadButton/touched}}. A [=placeholder axis=] MUST report <code>0</code>. [=Placeholder buttons=] and [=placeholder axis|axes=] MUST be omitted if they are the last element in the array or all following elements are also [=placeholder buttons=] or [=placeholder axis|axes=].
 
-Additional buttons or axes may be exposed after these reserved indices, and SHOULD appear in order of decreasing importance. Related axes (such both axes of a joystick) SHOULD be grouped and, if applicable, SHOULD appear in X, Y, Z order. If a device has both a touchpad and a joystick the UA MAY designate whichever it chooses to be the primary axis-based input. Buttons reserved by the UA or platform MUST NOT be exposed.
+Additional buttons or axes may be exposed after these reserved indices, and SHOULD appear in order of decreasing importance. Related axes (such both axes of a thumbstick) SHOULD be grouped and, if applicable, SHOULD appear in X, Y, Z order. Buttons reserved by the UA or platform MUST NOT be exposed.
```

Comment:
Can this text be MUST?

Reply:
I assume you're referring to the SHOULD, and the only reason I left it as such instead of MUST is because it may not always be apparent to the UA what that order would be.

Reply:
Oh, sorry I meant in this part:
> Related axes (such both axes of a thumbstick) SHOULD be grouped and, if applicable, SHOULD appear in X, Y, Z order.

Reply:
Got it! Done.